site/docs/id/guide/context.md

```diff
@@ -55,15 +54,17 @@ Object context selalu berisi informasi tentang bot-mu, yang dapat diakses melalu
 
 Ada sejumlah shortcut yang tersedia untuk object context.
 
-| Shortcut              | Deskripsi                                                                                  |
-| --------------------- | ------------------------------------------------------------------------------------------ |
-| `ctx.msg`             | Mendapatkan object message, termasuk yang sudah diedit                                     |
-| `ctx.chat`            | Mendapatkan object chat                                                                    |
-| `ctx.senderChat`      | Mendapatkan object chat pengirim dari `ctx.msg` (untuk pesan grup/channel anonim)          |
-| `ctx.from`            | Mendapatkan informasi penulis pesan, callback query, dan lainnya                           |
-| `ctx.inlineMessageId` | Mendapatkan id pesan inline dari callback query atau hasil inline yang dipilih             |
-| `ctx.entities`        | Mendapatkan entity pesan beserta teksnya, dapat disaring berdasarkan jenis entity-nya      |
-| `ctx.reactions`       | Mendapatkan reaksi dari suatu update [dengan mudah](./reactions#menyimak-perubahan-reaksi) |
+| Shortcut                   | Deskripsi                                                                                  |
+| -------------------------- | ------------------------------------------------------------------------------------------ |
+| `ctx.msg`                  | Mendapatkan object message, termasuk yang sudah diedit                                     |
+| `ctx.chat`                 | Mendapatkan object chat                                                                    |
+| `ctx.senderChat`           | Mendapatkan object chat pengirim dari `ctx.msg` (untuk pesan grup/channel anonim)          |
+| `ctx.from`                 | Mendapatkan informasi penulis pesan, callback query, dan lainnya                           |
+| `ctx.msgId`                | Mendapatkan id pesan untuk memproses pesan atau reaksi                                     |
```

Comment:
> Mendapatkan id pesan untuk **memproses** pesan atau reaksi

What does `memproses` do here?
So, `reaksi` need to add `memberi reaksi`?

Reply:
> But, `ctx.msgId` isn't a method right?

It is a getter, i.e. a property that will run some code when accessed. You do not have to do `ctx.msgId()`, and it is best to think about it as a regular property like `ctx.update`.

Reply:
Oh, I see...

Reply:
(in fact, all of the things listed in this table work that way, with the notable exceptions being `ctx.reactions()` and `ctx.entities()`

Reply:
I took a quick look at the code in `context.ts`, and sure enough it's a getter.

I never use getter or setter when creating classes. Thanks, I now understand when it needs to be used.